adam/ontology/phase1_ontology.py

```diff
@@ -147,6 +147,33 @@ def subtype(sub: OntologyNode, _super: OntologyNode) -> None:
 """
 subtype(GAZED_AT, PERCEIVABLE_PROPERTY)
 
+
+# Dowty's Proto-Roles: Issue 104; Dotwy, 91, page 572.
+# Agent Proto-Roles properties:
+AGENT_PROTO_ROLE = OntologyNode("agent-proto-role")
```

Comment:
We don't want to include the agent and patient proto-roles directly; rather the learner will figure them out from the atomic properties like `VOLITIONALLY_INVOLVED`, etc.